A/Antic/build_tarballs.jl

```diff
@@ -5,6 +5,24 @@ using BinaryBuilder, Pkg
 name = "Antic"
 version = v"0.2.3"
```

Comment:
You added an explanation for the version scheme, but are not really using it here